starter-project-web-vue2/pages/andualem/_id.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <nuxt-link style="text-decoration: none" :to="'/andualem/'">
+      <v-btn class="success">Back to Blogs</v-btn>
+    </nuxt-link>
+    <h2>Title: {{ title }}</h2>
+    <hr />
+    <h4>{{ content }}</h4>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+export default {
+  data() {
+    return {
+      content: '',
+      title: '',
+    }
+  },
+  async created() {
+    const config = {
+      headers: {
+        Accept: 'application/json',
+      },
+    }
+    try {
+      const res = await axios.get(
+        `https://blog-app-backend.onrender.com/api/articles/${this.$route.params.id}`,
```

Comment:
This section should be in a store file, and also no need to import axios, use this.$axios